reference/src/taimpl/src/internal/key_store.c

```diff
@@ -163,22 +164,24 @@ static bool write_to_storage(const void *data, const size_t length, const char *
         return false;
     }
 
-    // TODO SoC Vendor: replace this with code to write to secure storage (using TEE supplicant or other such method).
-    // The following code is for demonstration purposes of writing to disk using fopen/fwrite, and won't work in
-    // TEE environments.
-#if WRITE_TO_FILE_TEST
+    // TODO SoC Vendor: replace this with code to write to secure storage (using TEE
```

Comment:
I don't think we need to have the key written to secapi3 secure storage. Rather the SOC vendor needs to implement a function that passes the key data to the DRM TA corresponding to the key type (widevine, playready, netflix, etc.). It could coincide with writing data to secapi3 storage or some other mechanism. That is SOC vendor dependent and should be left up to them.